.github/workflows/ci.yml

```diff
@@ -8,16 +8,28 @@ on:
 
 jobs:
   build:
+    runs-on: ubuntu-latest
```

Comment:
question: why are we breaking out the build and node-compatibility jobs into separate jobs? Seems that they are the same and therefore redundant?

Reply:
So they are not quite the same. The problem we are trying to solve here is that someone with node 12 installed cannot _build_ the project. Vite requires node 18+ to build. But we still want to support someone with node 12 installing and using the _built_ version of the app.

So we now have two tasks:
- `build` will test that the project builds and the test passes, but runs only on node 22 (or whatever is latest)
- `node_compatibility` will check that the built output can still be imported and used on older node versions.

Reply:
I guess we could add the `node_compatibility` section after the build/test step of the `build` task 🤔 I could go either way

Reply:
Ah I see, nice! I'd maybe just clarify this as comments above each job?